extensions/oidc/deployment/src/main/java/io/quarkus/oidc/deployment/OidcBuildStep.java

```diff
@@ -482,6 +485,14 @@ public void registerHealthCheck(OidcBuildTimeConfig config, BuildProducer<Health
         }
     }
 
+    @Record(ExecutionTime.STATIC_INIT)
+    @BuildStep
+    FilterBuildItem registerBackChannelLogoutHandler(BeanContainerBuildItem beanContainerBuildItem, OidcRecorder recorder) {
+        Handler<RoutingContext> handler = recorder.getBackChannelLogoutHandler(beanContainerBuildItem.getValue());
+        int lastFilter = 1;
+        return new FilterBuildItem(handler, lastFilter);
```

Comment:
I haven't used this item before, what is the idea with the last fitter value being set to 1 ? I've started prototyping the protected resource metadata code which will also require a handler so I'm curious

Reply:
We use these filters for cors, auth, authz handlers, they apply on all paths (they are just route handlers without a path, the name is confusing really). Not sure about  a priority, I just need it to run after all before-mentioned, on second though 1 may be too strict, I'll just probably go with something like 50 to give others option to put further filters.

Reply:
AKA they are just normal route handlers applied here https://github.com/quarkusio/quarkus/blob/b72de630077cb1be66fe0a60282ea0bfa4fceb02/extensions/vertx-http/runtime/src/main/java/io/quarkus/vertx/http/runtime/VertxHttpRecorder.java#L434

Reply:
@michalvavrik On `main`, where dynamic tenants are not supported, we have path-specific back channel logout route handlers for static tenants. 
I wonder, can we optimize it somehow, having a wildcard back-channel handler called on every request, trying to make a match, can be costly I guess, especially given that the back-channel logout is probably in the 10% of the logout cases ? I wonder, is it possible somehow to have a Vert.x handler with a dynamic tenant's back-channel specific path, added at the request time, when the dynamic tenant is created ?

Reply:
I don't agree this addition is costly at all. You are mentioning "wildcard back-channel handler", but I simply thought adding validation is overkill. This PR doesn't change the original support for the path and matching exact paths is as fast as possible. I don't believe that Vert.X can have some more performant algorithm to match exact paths.

I suggest that I just add a validation that fails if a wildcard is detect and that is it.

Reply:
I have added an exception raised when the back-channel logout path contains `*`. Now we support only exact paths (like before). The exact paths are simply checked by `exactPathMatches.get(path)` (the ending (or absence of it) slash is taken into consideration) so I think unless there is a hundred of tenants with such a path, it is not an issue.